source/index.rst

```diff
@@ -53,6 +65,10 @@ The National Renewable Energy Laboratory is a national laboratory of the U.S. De
 
 Sandia National Laboratories is a multi-mission laboratory managed and operated by National Technology and Engineering Solutions of Sandia, LLC., a wholly owned subsidiary of Honeywell International, Inc., for the U.S. Department of Energy’s National Nuclear Security Administration under contract DE-NA0003525.
 
+.. Note:: 
+	Adam:
+	This information is repeated from the acknowledgements section.
+
```

Comment:
I think we could move it from acknowledgements and keep it here... or honestly it doesn't hurt to over-reference your funding.